lib/AppInfo/Application.php

```diff
@@ -227,6 +229,10 @@ public function register(IRegistrationContext $context): void {
 		$context->registerMiddleware(AuthorizedAdminSettingMiddleware::class);
 	}
 
+	/**
+	 * @throws ContainerExceptionInterface
+	 * @throws Throwable
```

Comment:
When does that throw Throwable? For me that should never be documented, Throwable which are not Exception are bugs to be fixed, not documented. Anything can throw a Throwable if you use it wrong.
Would also be good if those `@throws` tag would include a description of when to expect a throw.